protocol/json-schemas/tbdex-message-schema.json

```diff
@@ -0,0 +1,276 @@
+{
+    "definitions": {
+        "metadata": {
+            "type": "object",
+            "properties": {
```

Comment:
missing `type` property? 
<img width="949" alt="image" src="https://github.com/TBD54566975/tbdex-protocol/assets/118510103/f428fb5d-3481-41a0-ab3e-12b8a9daa194">

Reply:
nvm i think i see how you did it below

```"properties": {
                "type": {
                    "enum": [
                        "offering",
                        "rfq",
                        "quote",
                        "order",
                        "orderStatus"
                    ]
                }
            }```